app/views/spree/orders/form/_update_buttons.html.haml

```diff
@@ -1,7 +1,7 @@
 .row
   .columns.small-12.medium-3
-    - if current_order.andand.distributor == @order.distributor
```

Comment:
Do you have any idea what is this actually checking @oeoeaio regarding `distributor`? According to [the definition of current_order](https://github.com/coopdevs/spree/blob/spree-upgrade-step-6/core/lib/spree/core/controller_helpers/order.rb#L18), this method is meant to be used while completing the order and it'll only return it if it's not completed yet.

In this case it is since we just finished the purchase and also `@order` and `current_order` would represent the same object.